core-bundle/src/Resources/contao/library/Contao/Database/Installer.php

```diff
@@ -369,7 +369,7 @@ public function getFromDb()
 					// Field type
 					if ($field['length'])
 					{
-						$field['type'] .= '(' . $field['length'] . ($field['precision'] ? ',' . $field['precision'] : '') . ')';
+						$field['type'] .= '(' . $field['length'] . (isset($field['precision']) ? ',' . $field['precision'] : '') . ')';
```

Comment:
```suggestion
						$field['type'] .= '(' . $field['length'] . (!empty($field['precision']) ? ',' . $field['precision'] : '') . ')';
```

Might be more safe?

Reply:
I thought about this as well, but 
 1. we use `isset` for other such cases
 2. `empty` would return false for `0`. Even though that's probably an invalid value, we don't really want to validate the value itself but if something is set we want to use it?